src/main/java/sirius/web/security/oauth/ReceivedTokens.java

```diff
@@ -22,10 +22,12 @@
  * @param accessToken           the access token received from the authorization server
  * @param refreshToken          the refresh token received from the authorization server
  * @param type                  the type of the tokens received from the authorization server, e.g. "Bearer" or "MAC"
+ * @param accessTokenExpiresAt  the date at which the access token expires, might be null if the server response
+ *                              contains no information
  * @param refreshTokenExpiresAt the date at which the refresh token expires, might be estimated if no JWT bearer
  *                              token is given
  */
-public record ReceivedTokens(String accessToken, String refreshToken, String type,
+public record ReceivedTokens(String accessToken, String refreshToken, String type, LocalDateTime accessTokenExpiresAt,
                              LocalDateTime refreshTokenExpiresAt) {
```

Comment:
**Optional:** This formatting of the constructor parameters looks a bit weird. Is this autoformatted with our default formatter settings?

Reply:
This is autoformatted. I cannot see an issue on my formatter settings. I case it is wrong, please ping me.